localMatchApp/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
+
+PROJECT(local_match
+	LANGUAGES CXX)
+
+# Find includes in corresponding build directories
+SET(CMAKE_INCLUDE_CURRENT_DIR ON)
+# Instruct CMake to run moc automatically when needed
+SET(CMAKE_AUTOMOC ON)
+# Create code from a list of Qt designer ui files
+SET(CMAKE_AUTOUIC OFF)
+
+# Find the QtWidgets library
+FIND_PACKAGE(Qt5Widgets CONFIG REQUIRED)
+
+INCLUDE_DIRECTORIES(sources headers)
+
+FILE(GLOB code_SOURCES
```

Comment:
Add a comment describing what this does. Same with the rest of the lines in the CMake